account_invoice_move_currency/README.rst

```diff
@@ -15,25 +15,32 @@ Account Invoice Move Currency
 =============================
 
 This module extends the functionality of account invoice to
-allows force the seat on an invoice generated save the data in another currency.
+allows force the seat on an invoice generated save the data in another currency. This module adds funcionality to save the secondary currency in the wizard "change" that introduce the module "Account Usability"
```

Comment:
* "the seat" is bad english to refer to "el asiento", should be journal.entries
- "generated save the data", I don not understand what it means here we have save verb without sustantive.
- account usability is accout ux.